test/test_mpi_periodic_elastic_half_space.py

```diff
@@ -133,21 +133,21 @@ def test_sineWave_disp(comm, pnp, nx, ny, basenpoints):
         fftengine.create_plan(1)
 
         kpressure = substrate.evaluate_k_force(
-            disp[substrate.subdomain_slices]) / substrate.area_per_pt / (
-                            nx * ny)
+            disp[substrate.subdomain_slices]) / substrate.area_per_pt #/ (
```

Comment:
@sindhu2993  Here you just made the normalisation more logical, correct ? The test is exactly the same since you changed the reference values accordingly.

Reply:
yes.